.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,11 +9,15 @@ assignees: ''
 
 ---
 
+Download and run checkstyle cli: https://checkstyle.org/cmdline.html#Download_and_Run
```

Comment:
> .github/ISSUE_TEMPLATE/bug_report.md

main word here is "TEMPLATE", template means that all text will be text of issue author, as he did wrote it himself.
Template means that all text should not be removed and user just need to fill gaps.

Reply:
Got it.